screenpipe-app-tauri/components/search-chat.tsx

```diff
@@ -81,13 +81,84 @@ import { Popover, PopoverContent, PopoverTrigger } from "./ui/popover";
 import { useHealthCheck } from "@/lib/hooks/use-health-check";
 import { SearchHistory } from "@/lib/types/history";
 
+
+    
 interface SearchChatProps {
   currentSearchId: string | null;
   onAddSearch: (searchParams: any, results: any[]) => Promise<string>;
   searches: SearchHistory[];
 }
 
-export function SearchChat({ currentSearchId, onAddSearch, searches }: SearchChatProps) {
+export function SearchChat({ currentSearchId, onAddSearch, searches }: SearchChatProps)) {
+  
+  interface Agent {
```

Comment:
@onyedikachi-david merge broke

it does not compile at all

Reply:
Yes, it's in the merge commit. The chat component has double bracket, made that error during merge conflict resolution. Fixed it locally, was about pushing before I got engaged with something else. Will fix it ASAP.